grpc_opa/authorizer.go

```diff
@@ -252,6 +258,8 @@ func (a *DefaultAuthorizer) Evaluate(ctx context.Context, fullMethod string, grp
 	}
 
 	if !opaResp.Allow() {
+		forbidReason := opaResp.rbacForbiddenReason()
+		logger.Infof("Request forbidden because these RBAC checks failed:%s", forbidReason)
 		return false, ctx, ErrForbidden
```

Comment:
Another way to surface error messages is to wrap them in the error being returned.
```suggestion
		return false, ctx, fmt.Errorf("%w: %s", ErrForbidden, forbidReason)
```

Caller verifies type of error with `errors.Is(err, grpc_opa.ErrForbidden)`